playstore/validator.go

```diff
@@ -44,14 +37,17 @@ type Client struct {
 // New returns http client which includes the credentials to access androidpublisher API.
 // You should create a service account for your project at
 // https://console.developers.google.com and download a JSON key file to set this argument.
-func New(jsonKey []byte) (Client, error) {
-	ctx := context.WithValue(oauth2.NoContext, oauth2.HTTPClient, &http.Client{
-		Timeout: timeout,
+func New(ctx context.Context, jsonKey []byte) (Client, error) {
+	return NewWithClient(ctx, jsonKey, &http.Client{
+		Timeout: defaultTimeout,
 	})
+}
 
+// NewWithClient returns http client which includes custom internal http client and the credentials to access androidpublisher API.
+func NewWithClient(ctx context.Context, jsonKey []byte, cli *http.Client) (Client, error) {
```

Comment:
You embed the given context in a client.
But context is different in each request.
It is not safe for concurrent use, right?

Reply:
I'm sorry that I don't know the GAE problem in detail because I'm using this library on AWS.

GAE requires a special urlfetch.Transport in a document.
https://cloud.google.com/appengine/docs/standard/go/urlfetch/reference#Client

So you should create a urlfetch.Client which includes gae.Context.
Why do you add context argument?
Doesn't the following code work?
```
func NewWithClient(jsonKey []byte, cli *http.Client) {
        c := context.WithValue(context.Background(), oauth2.HTTPClient, cli)
        conf, err := google.JWTConfigFromJSON(jsonKey, androidpublisher.AndroidpublisherScope)
        return Client{conf.Client(c)}, err
}

func main() {
        c := appengine.NewContext(r)
        httpCli := urlfetch.Client(c)
        cli := NewWithClient(jsonKey, httpCli)
}
```

Reply:
I've send a new pull-request #64.
Could you check if this code and test resolves this problem or not.